src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -161,6 +159,12 @@ def get_sdk(self):
             self._sdk = [{"sdk": normalized_sdk_tag_from_event(self.event)}]
         return self._sdk
 
+    def get_family(self):
+        self._family = self._family or [
+            {"family": get_behavior_family_for_platform(self.event.get("platform"))}
+        ]
```

Comment:
Used this instead of an `if` as it works equally well, but CodeCov considers this line fully covered.